runtime/services/otad/step.js

```diff
@@ -68,8 +68,9 @@ module.exports.downloadImage = downloadImage
 function downloadImage (delegate, info, callback) {
   var dest = persistance.getImagePath(info)
   compose([
-    cb => {
-      checkDiskAvailability(info.imageSize, dest, cb)
+    cb => wget.fetchImageSize(info.imageUrl, cb),
+    (cb, imageSize) => {
```

Comment:
What's the use of first param `cb`? I notice you put `null` in when invoke this callback.

Reply:
Its signature is same as common Node.js callback `function (err?: Error, res?: any)`.